docs/releases/release2101.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 99.84
+---
+
+# Release 2.10.1
+
+Date: 6th of June, 2025
```

Comment:
```suggestion
Date: June 6, 2025
```

May be we should update all the dates in this format. Don't know how we missed this previously.